lib/Voxa-Opearlo.js

```diff
@@ -58,7 +58,18 @@ function register(skill, config) {
     OpearloAnalytics.registerVoiceEvent(alexaEvent.session.user.userId,
       'Custom',
       'Session ended');
-    debug('SessionEndedRequest registered');
+    debug('SessionEndedRequest logged');
+
+    if(alexaEvent.request.type == 'SessionEndedRequest') { //If we get an explict session ended request, we'll need to trigger this ourselves
+      return new Promise( (resolve,reject) =>  OpearloAnalytics.recordAnalytics(
```

Comment:
Should we promisify `OpearloAnalytics.recordAnalytics` to avoid resolve and reject here?